pkgs/development/python-modules/libusb-package/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  libusb1,
+  pytestCheckHook,
+  python,
+  pythonOlder,
+  setuptools-scm,
+  importlib-resources,
+  pyusb,
+}:
+
+buildPythonPackage rec {
+  pname = "libusb-package";
+  version = "1.0.26.2";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "pyocd";
+    repo = "libusb-package";
+    rev = "v${version}";
+    hash = "sha256-bM+v3eeQQHiBKsFky51KdDYlRjFzykQFkuFKkluqkjY=";
+  };
+
+  patches = [ ./fix-pytest.patch ];
```

Comment:
Could this patch get a more descriptive name or commit message, detailing why it's there and the intended effect?